projects/packages/action-bar/package.json

```diff
@@ -0,0 +1,49 @@
+{
+	"private": true,
+	"name": "@automattic/jetpack-action-bar",
+	"version": "0.1.0",
+	"description": "An easy way for visitors to follow, like, and comment on your WordPress site.",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/packages/action-bar/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[Package] Action Bar"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/packages/jetpack-action-bar"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "NODE_PATH=\"$PWD/node_modules\" pnpm run clean && webpack --config ./webpack.config.action-bar.js",
+		"build-production": "NODE_ENV=production BABEL_ENV=production pnpm run build && pnpm run validate",
+		"clean": "rm -rf build/ .cache/",
+		"validate": "pnpm exec validate-es --no-error-on-unmatched-pattern build/",
+		"watch": "pnpm run build --watch"
+	},
+	"engines": {
+		"node": "^16.13.2",
+		"yarn": "use pnpm instead - see docs/yarn-upgrade.md"
+	},
+	"dependencies": {
+		"@wordpress/i18n": "4.12.0",
+		"preact": "10.5.15"
```

Comment:
> Is this because the feature will be used on the frontend, and we're looking for a small library to use on the frontend?

Yes, exactly. I'm trying to minimize the size for something that will potentially load for every single post across WP.com. We're still exploring what's the best approach--I'm hesitant to load all of react and react-dom, but we'll do some testing to see the actual size difference.